controllers/articles.controllers.js

```diff
@@ -70,4 +69,19 @@ function getArticle(req, res, next) {
     });
   }
 
-module.exports = { getArticle, getAllArticles, getAllCommentsByArticle, postComment, patchArticle }
\ No newline at end of file
+  function deleteComment(req, res, next) {
+    const { comment_id } = req.params;
+    return removeComment(comment_id)
+    .then((comment) => {
+      res.status(204).send({comment})
+    })
+    .catch((err) => {
+      console.log(err)
+      if(err.code === '22P02') {
+        res.status(400).send({ msg : 'bad request'})
+      }
+      next(err);
+    });
```

Comment:
This is working to handle the PSQL error but it isn't the most effect place for the `if (err.code === '22P02')` to be
This logic is best placed in your error handling middleware in your `app.js` file, otherwise you would need to have this same `if` statement in multiple controllers rather than just having it once in `app.js`